src/main/webapp/js/maps-page-loader.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * To style the map
```

Comment:
A little ambiguous. What does style the map mean? Which map? And what kind of style? Ideally it should be clear what the function does from reading this, so that I know when I should use this function, to do what.